src/applications/personalization/profile-2/sass/mobile-sidenav-trigger.scss

```diff
@@ -7,6 +7,9 @@
     width: 100%;
     margin: 0;
     border-radius: 0;
+    display: flex;
```

Comment:
we were getting this behavior from sitewide styles applied to the `va-btn-sidebarnav-trigger` class